lib/server.rb

```diff
@@ -51,7 +52,12 @@ class Server < Grape::API
       desc "Read a form."
       get do
         repository = Repositories::FormsRepository.new(@database)
-        repository.get(params[:form_id])
+        page_repository = Repositories::PagesRepository.new(@database)
+
+        form = repository.get(params[:form_id])
+        pages = page_repository.get_pages_in_form(params[:form_id]).sort_by { |page| page[:id] }
+        form[:start_page] = pages.first[:id]
```

Comment:
I think this fails with a 500 when the form doesn't have any pages which causes the runner to fail too.